tests/literals.pyi

```diff
@@ -1,5 +1,12 @@
 from typing import Literal
 
 Literal[None]  # Y061 None inside "Literal[]" expression. Replace with "None"
+Literal[None, None]  # Y061 None inside "Literal[]" expression. Replace with "None"
 Literal[True, None]  # Y061 None inside "Literal[]" expression. Replace with "Literal[True] | None"
 Literal[1, None, "foo", None]  # Y061 None inside "Literal[]" expression. Replace with "Literal[1, 'foo'] | None"
+
+Literal[True, True]  # Y062 Duplicate "Literal[]" member "True"
+Literal[True, True, True]  # Y062 Duplicate "Literal[]" member "True"
+Literal[True, False, True, False]  # Y062 Duplicate "Literal[]" member "True" # Y062 Duplicate "Literal[]" member "False"
+
+Literal[None, True, None, True]  # Y061 None inside "Literal[]" expression. Replace with "Literal[True, True] | None" # Y062 Duplicate "Literal[]" member "True"
```

Comment:
yeah, I'm still not a fan of this :( I think we should just not emit Y061 if we're going to emit Y062. We can add a note to the Y061 docs in `ERRORCODES.md` stating this behaviour. We already have a few other codes that interact with each other in slightly subtle ways like this, and I think it's better than suggesting code that we'd then immediately complain about.

I'd go for something like this (the diff is relative to your PR branch)

```diff
diff --git a/pyi.py b/pyi.py
index a39ee1a..3fc7cb2 100644
--- a/pyi.py
+++ b/pyi.py
@@ -1474,21 +1474,24 @@ class PyiVisitor(ast.NodeVisitor):
     def _visit_typing_Literal(self, node: ast.Subscript) -> None:
         analysis = _analyse_typing_Literal(node)

-        if analysis.contains_only_none:
-            self.error(node.slice, Y061.format(suggestion="None"))
-        elif analysis.none_members:
-            if len(analysis.members_without_none) == 1:
-                new_literal_slice = unparse(analysis.members_without_none[0])
-            else:
-                new_slice_node = ast.Tuple(elts=analysis.members_without_none)
-                new_literal_slice = unparse(new_slice_node).strip("()")
-            suggestion = f"Literal[{new_literal_slice}] | None"
-            self.error(analysis.none_members[0], Y061.format(suggestion=suggestion))
-
+        Y062_encountered = False
         for member_list in analysis.members_by_dump.values():
             if len(member_list) > 1 and not _is_None(member_list[0]):
+                Y062_encountered = True
                 self.error(member_list[1], Y062.format(unparse(member_list[1])))

+        if not Y062_encountered:
+            if analysis.contains_only_none:
+                self.error(node.slice, Y061.format(suggestion="None"))
+            elif analysis.none_members:
+                if len(analysis.members_without_none) == 1:
+                    new_literal_slice = unparse(analysis.members_without_none[0])
+                else:
+                    new_slice_node = ast.Tuple(elts=analysis.members_without_none)
+                    new_literal_slice = unparse(new_slice_node).strip("()")
+                suggestion = f"Literal[{new_literal_slice}] | None"
+                self.error(analysis.none_members[0], Y061.format(suggestion=suggestion))
+
         self.visit(node.slice)

     def _visit_slice_tuple(self, node: ast.Tuple, parent: str | None) -> None:
```